src/classes/models/database.py

```
@@ -0,0 +1,247 @@
+        """
+        conn = None
+        try:
+            conn = self.get_connection()
+            with conn.cursor() as cursor:
+                cursor.execute(query, params)
+                
+                if query.strip().upper().startswith('SELECT'):
+                    if fetch_one:
+                        result = cursor.fetchone()
+                    else:
+                        result = cursor.fetchall()
```

Ruff (0.8.2)

Use ternary operator `result = cursor.fetchone() if fetch_one else cursor.fetchall()` instead of `if`-`else`-block

Replace `if`-`else`-block with `result = cursor.fetchone() if fetch_one else cursor.fetchall()`

(SIM108)